annotations/halkyon-annotations/src/main/java/io/dekorate/halkyon/annotation/HalkyonCapabilities.java

```diff
@@ -0,0 +1,23 @@
+package io.dekorate.halkyon.annotation;
+
+import io.dekorate.kubernetes.config.Configuration;
+import io.sundr.builder.annotations.Adapter;
+import io.sundr.builder.annotations.Buildable;
+import io.sundr.builder.annotations.Pojo;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Buildable(builderPackage = "io.dekorate.deps.kubernetes.api.builder")
+@Pojo(name = "CapabilitiesConfig", mutable = true, superClass = Configuration.class, relativePath = "../config", withStaticAdapterMethod = false,
+  adapter = @Adapter(suffix = "Adapter", relativePath = "../adapter", withMapAdapterMethod = true))
+@Target({ElementType.CONSTRUCTOR, ElementType.TYPE})
+@Retention(RetentionPolicy.SOURCE)
+public @interface HalkyonCapabilities {
```

Comment:
Is this annotation still needed if you put its element on HalkyonComponent?

Reply:
Not needed anymor, we can remove it.